### PR TITLE
Replace Restler HTTP library with Request

### PR DESCRIPTION
**Pull request by jarrmill**
## Description of the change

Relates to [Marketo timeout bug to ticket.](https://app.shortcut.com/active-prospect/story/27491/marketo-create-lead-integration-errors-out)

This is 1 of 2 Marketo related PRs, So see the other, click [here](https://github.com/activeprospect/leadconduit-integration-marketo/pull/42)

This PR pulls out the long-deprecated Restler HTTP library, and replaces it with Request. On my local environment I have validated with all 9 Marketo integrations we offer that all are functioning with these changes. If this is approved and we can get it on dev/staging, I can verify it there as well.

Please let me know if you have any questions, and thanks all for your patience and help.

Lastly... 68 additions and 794 deletions 😈 

## Type of change
- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change
- [x] Technical Debt
- [ ] Documentation

## Related tickets

https://app.shortcut.com/active-prospect/story/27491/marketo-create-lead-integration-errors-out

### Development and Testing

- [ ]  Lint rules pass locally.
- [ ]  The code changed/added as part of this pull request has been covered with tests, or this PR does not alter production code.
- [x]  All tests related to the changed code pass in development, or tests are not applicable.

### Code Review

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached.
- [ ]  At least two engineers have been added as "Reviewers" on the pull request.
- [ ]  Changes have been reviewed by at least two other engineers who did not write the code.
- [ ]  This branch has been rebased off master to be current.

### Tracking 
- [ ]  Issue from Clubhouse has a link to this pull request.
- [ ]  This PR has a link to the issue in Clubhouse.

### QA
- [ ]  This branch has been deployed to staging and tested.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,10 +17,10 @@
     "backoff": "2.4.x",
     "bluebird": "2.3.x",
     "bunyan": "1.0.x",
-    "iconv-lite": "0.2.11",
     "lodash": "^4.17.21",
     "moment": "2.24.0",
-    "qs": "6.9.x"
+    "request": "^2.88.2",
+    "request-promise-native": "^1.0.9"
```
Comment (cgrayson) [inline package.json]: I didn't see where this is used.

**Review by cgrayson — DISMISSED** [review]
:+1:

**Review by st3v3odom — DISMISSED** [review]
LGTM, besides the unused package reference Chris pointed out.

**Review by st3v3odom — APPROVED** [review]
LGTM

**Pull request merged**